docs/3-KnowledgeBase.stories.mdx

```diff
@@ -219,4 +220,176 @@ This would then be the result:
+              name="country"
+              value={values.country}
+              onChange={(e) =>
+                setValues({
+                  ...values,
+                  country: e.detail.selectedOption.innerText
+                })
+              }
+            >
+              <Option>Germany</Option>
+              <Option>France</Option>
+              <Option>Italy</Option>
+            </Select>
+          </FormItem>
+          <FormItem label={'Date of Birth'}>
+            <DatePicker
+              value={values.birthday}
+              onChange={(e) => {
+                setValues({
+                  ...values,
+                  birthday: e.detail.value
+                });
+              }}
+            />
+          </FormItem>
+          <FormItem label={'Payment methods'}>
+            <MultiComboBox onSelectionChange={handleSelectionChange}>
+              <MultiComboBoxItem text="Credit card" />
+              <MultiComboBoxItem text="PayPal" />
+              <MultiComboBoxItem text="Bank transfer" />
+            </MultiComboBox>
+          </FormItem>
+          <FormItem>
+            <CheckBox
+              required
+              name="terms"
+              text="I accept the terms of service"
+              value={values.terms}
+              onChange={(e) =>
+                setValues({
+                  ...values,
+                  terms: e.target.checked
+                })
+              }
+            />
+          </FormItem>
+          <FormItem>
+            <Button onClick={handleSubmit}>Submit</Button>
+          </FormItem>
+        </FormGroup>
+      </Form>
+    </ThemeProvider>
+  );
+}
+```
+
+</details>
+
+Now the `AccountForm` would compile to this:
+
+<CreateAccountForm />
+
+Note: Whenever a component has a custom event (e.g. onChange event from `DatePicker`), the corresponding value should be fetched via event.detail (not event.target). This will give you the correct updated value since the internal input has already been updated. At the time the event is fired, however, it is not yet updated and therefore event.target might also not be updated yet.
+
+### #2 Example with UI5 Web Components for React and Formik + Yup
+
+Input components from UI5 Web Components for React can be used with external libraries like Formik and Yup.
+Formik is an open source form library for React and React Native (https://formik.org/).
+Yup is a schema builder for runtime value parsing and validation (https://github.com/jquense/yup).
+
+Formik keeps track of your form's state and then exposes it plus a few reusable methods and event handlers (handleChange, handleBlur, and handleSubmit) to your form via props
+It comes with a few extra components to make life easier and less verbose like `Form`, `Field`, and `ErrorMessage`.
+Also Formik has a special config option for Yup called validationSchema.
+
+To inject an input component from UI5 Web Components for React the `Field` component has a prop called `as`.
```

Comment:
Please also mention that Formik offers a "headless" solution, which is using a similar approach to this one, but is not explained in detail here.